dm/worker/worker.go

```diff
@@ -406,12 +407,12 @@ func (w *Worker) doFetchDDLInfo(ctx context.Context, ch chan<- *pb.DDLInfo) {
 		return // should not go here
 	}
 
-	w.RLock()
-	w.subTasks[v.Task].SaveDDLInfo(v)
```

Comment:
does this mean if v.Task not in `w.subTasks`, it will panic in the old code, do we need to add a log if `st` is nil

Reply:
YES, in the old code, if calling this method when closing the Worker, it may panic.

added a log in https://github.com/pingcap/dm/pull/259/commits/6905cdd2a1123408668f196076e33e8f2ede80e1.